src/qibocal/protocols/characterization/rabi/utils.py

```diff
@@ -213,4 +213,4 @@ def period_correction_factor(phase: float):
 
     """
 
-    return 1 - np.modf(phase / np.pi)[0]
+    return np.round(1 + phase / np.pi) - phase / np.pi
```

Comment:
```suggestion
    x = phase / np.pi
    return np.round(1 + x) - x
```
Any better name than `x` is a further improvement.